keps/sig-api-machinery/20190724-http-status-check.md

```diff
@@ -0,0 +1,204 @@
+---
+title: http probe enhancement
+authors:
+  - "@konghui"
+owning-sig: sig-apimachinery
```

Comment:
@konghui Based on @liggitt [comment](https://github.com/kubernetes/kubernetes/pull/80411#issuecomment-541703286) it seems like this KEP belongs to sig-node. As far I understand, we need to rename a file, change `owning-sig` and fill out `reviewers`/`approvers` sections.

Reply:
done